Examples/Fermions/fermi_hubbard_slater.py

```diff
@@ -21,14 +21,14 @@
 n_sites = g.n_nodes
 
 # create a hilbert space with 2 up and 2 down spins
-hi = nkx.hilbert.SpinOrbitalFermions(n_sites, s=1 / 2, n_fermions=(2, 2))
+hi = nkx.hilbert.SpinOrbitalFermions(n_sites, s=1 / 2, n_fermions_per_spin=(2, 2))
```

Comment:
if we want to be even clearer, and more consistent, then let's also do `n_spin_states=2` instead of `s=1 /2` in the constructor. the same should be done for Spin